mainmatter/introduction/graphics_apis/index.tex

```diff
@@ -1,33 +1,108 @@
+The state includes, amongst others, the compiled shaders, textures and allocated buffers.
+By keeping it implicit in Direct3D 11, the exact render state is only known when the draw call is made, meaning that the \gls{GPU} has no opportunity to pre-compute any commands in preparation.
+In Direct3D 12, the state is made explicit with a \glspl{PSO}, which contains all the state information.
+As each command list is associated with a \gls{PSO}, the \gls{GPU} may do pre-computations to gain performance.
+A downside is that each \gls{PSO} must be specified at compile time, meaning that the developer must have some foresight regarding the states being used.
+Yet, this allows for \gls{GPU} instructions and state to be set up at compile time, resulting in fast switches between states at runtime.  
+
+While Direct3D 12 builds on top of Direct3D 11, Khronos decided to create a whole new \gls{API} from scratch for very low-level graphics programming, Vulkan.
+This \gls{API} was released in early 2016 to compete with Direct3D 12.
+From the official overview released by Khronos at launch, Vulkan seems to follow the same design philosophy as Direct3D 12 \cite{vulkanPresentationFeb2016}.
+That is, making the developer responsible for more tasks in return for more flexibility.
+It too aims to lower driver overhead, allows for asynchronous draw calls, pushing computation to compile time, and making the render state explicit.
+
+Like OpenGL, Vulkan also focuses on cross-platform support.
+As of October 2017, it now supports Android 7, Windows 7, 8 \& 10 as well as Ubuntu\cite{vulkanConformance}.
+The support for older versions of windows is important, as according to the steam survey referred to in the last subsection, only ~46\% of steam users have windows 10 installed out of the ~97\% that run Windows\cite{steamStats}.
+Especially Windows 7 remains popular at ~44\%. Interestingly enough, according to the survey ~84\% of users have modern \glspl{GPU} with Direct3D 12 support.
+This means that there is a sizeable portion of non-windows 10 users, who need to turn to Vulkan to get the best performance out of their modern \glspl{GPU} on newer games.
+Vulkan seems to be up for the task, already being supported on many \glspl{GPU} \cite{vulkanConformance}.
+Additionally, companies like Valve have started backing Vulkan, because of its support for older windows versions along with Unix-based systems \cite{siggraph2015}.
+The amount of games supported by Direct3D 12 and Vulkan is also about the same according to Wikipedia listings, at about 30 games each \gls{API} \cite{wikiVulkanGames} \cite{wikiDX12Games}.
+
+Yet, the small amount of games in each case suggests a slow adoption of both \glspl{API}.
+This may be because the tools are at a very low-level of abstraction, and the performance that they grant is only needed for certain intensive games.
```

Comment:
Additionally, it could also be because most AAA games take years to develop. Both API's are fairly new. You dont switch to a newer GPU driver in the middle of production or almost towards the end, that's the kind of decision that are made pre-production and then never changed.